content/en/platform/corda/4.9/community/release-notes.md

```diff
@@ -17,6 +17,22 @@ tags:
 
 # Corda Community Edition 4.9 release notes
 
+## Corda Community Edition 4.9.7 release notes
+
+Corda Community Edition 4.9.7 is a patch release of Corda Community Edition which includes resolved issues.
+
+### Fixed issues
+
+* Corda provides the NodeDriver to help developers write integration tests. Using the NodeDriver, developers can bring up nodes locally to run flows and inspect state updates. Previously, there was an issue with build pipelines with tests failing, as on some occasions, notaries took more than one minute (the default timeout value) to start.
+
+  To resolve this, the NodeDriver now has a new parameter, notaryHandleTimeout. This parameter specifies how long to wait (in minutes) for a notary handle to come back after the notary has been started.
```

Comment:
Can you remove (in minutes).

Reason:
Its a Duration type, so user can specify themselves the unit.